src/api/v1beta1/feature_flags.go

```diff
@@ -31,12 +31,13 @@ const (
 	AnnotationFeaturePrefix = "feature.dynatrace.com/"
 
 	// activeGate
-	AnnotationFeatureDisableActiveGateUpdates         = AnnotationFeaturePrefix + "disable-activegate-updates"
-	AnnotationFeatureDisableActiveGateRawImage        = AnnotationFeaturePrefix + "disable-activegate-raw-image"
-	AnnotationFeatureActiveGateAppArmor               = AnnotationFeaturePrefix + "activegate-apparmor"
-	AnnotationFeatureActiveGateReadOnlyFilesystem     = AnnotationFeaturePrefix + "activegate-readonly-fs"
-	AnnotationFeatureAutomaticKubernetesApiMonitoring = AnnotationFeaturePrefix + "automatic-kubernetes-api-monitoring"
-	AnnotationFeatureActiveGateIgnoreProxy            = AnnotationFeaturePrefix + "activegate-ignore-proxy"
+	AnnotationFeatureDisableActiveGateUpdates              = AnnotationFeaturePrefix + "disable-activegate-updates"
+	AnnotationFeatureDisableActiveGateRawImage             = AnnotationFeaturePrefix + "disable-activegate-raw-image"
+	AnnotationFeatureActiveGateAppArmor                    = AnnotationFeaturePrefix + "activegate-apparmor"
+	AnnotationFeatureActiveGateReadOnlyFilesystem          = AnnotationFeaturePrefix + "activegate-readonly-fs"
+	AnnotationFeatureAutomaticK8sApiMonitoring             = AnnotationFeaturePrefix + "automatic-kubernetes-api-monitoring"
+	AnnotationFeatureAutomaticK8sApiMonitoringClusterLabel = AnnotationFeaturePrefix + "automatic-kubernetes-api-monitoring-cluster-label"
```

Comment:
```suggestion
	AnnotationFeatureAutomaticK8sApiMonitoringClusterLabel = AnnotationFeaturePrefix + "automatic-kubernetes-api-monitoring-name"
```

Reply:
Can you provide some justification? It's not "monitoring name". REST method used under the hood defines it as a "label".

Reply:
the whole feature flag is already too long, just `cluster` is also fine with me